I get a test error and from this discussion: https://forum.freecadweb.org/viewtopic.php?f=3&t=69276&hilit=coin.cast&start=20#p602100 it seems to be due to pivy 0.6.7 so it might be better to roll that one back for now.
```
======================================================================
ERROR: testRayPick (MeshTestsApp.PivyTestCases)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/freecad/Mod/Mesh/MeshTestsApp.py", line 447, in testRayPick
    det=coin.cast(det,str(det.getTypeId().getName()))
SystemError: <built-in function cast> returned NULL without setting an error

----------------------------------------------------------------------
Ran 776 tests in 148.494s

FAILED (errors=1)
System exit

```

--- a/org.freecadweb.FreeCAD.yaml
+++ b/org.freecadweb.FreeCAD.yaml
@@ -221,10 +221,13 @@ modules:
     build-commands:
       - python3 setup.py build
       - python3 setup.py install --prefix=/app
+    build-options:
+      env:
+        PREFIX: /app
     sources:
       - type: archive
-        url: https://github.com/coin3d/pivy/archive/0.6.7.tar.gz
-        sha256: 37e33d85117aac27640e011df74ddcd77f270428300df916b46ee5c50645d582
+        url: https://github.com/coin3d/pivy/archive/0.6.5.tar.gz
+        sha256: 16f2e339e5c59a6438266abe491013a20f53267e596850efad1559564a2c1719
       - type: shell
         commands:
           - sed -i 's|QUIET|REQUIRED|g' CMakeLists.txt

--- a/org.freecadweb.FreeCAD.yaml
+++ b/org.freecadweb.FreeCAD.yaml
@@ -704,7 +704,8 @@ modules:
     sources:
       - type: git
         url: https://github.com/FreeCAD/FreeCAD.git
-        tag: '0.20beta1'
+        tag: '0.20'
+        commit: 68e337670e227889217652ddac593c93b5e8dc94
       - type: file
         path: openscad.sh
       # we need this patch so that FreeCAD can find QAssistant
@@ -716,7 +717,7 @@ modules:
           # to obtain the revcount for a given gitRef (commit hash, branch or tag) run
           # echo $(($(curl -s 'https://api.github.com/repos/FreeCAD/FreeCAD/compare/120ca87015...'"$gitRef"'' | grep "ahead_by" | sed -s 's/ //g;s/"ahead_by"://;s/,//')+1))
           - |
-            revcount=28936
+            revcount=29177
             sed -i 's|${PACKAGE_WCREF}|'"$revcount"' (Git)|' src/Build/Version.h.cmake
             sed -i 's|@PACKAGE_VERSION@|@PACKAGE_VERSION@.'"$revcount"'|' src/XDGData/org.freecadweb.FreeCAD.appdata.xml.in
           - sed -i 's|${PACKAGE_WCURL}|https://github.com/FreeCAD/FreeCAD.git|' src/Build/Version.h.cmake

